src/main/java/frc/robot/subsystems/ExampleSubsystem.java

```diff
@@ -7,14 +7,23 @@
 
 package frc.robot.subsystems;
 
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+
+import edu.wpi.first.wpilibj.Joystick;
 import edu.wpi.first.wpilibj.command.Subsystem;
 
 /**
  * An example subsystem.  You can replace me with your own Subsystem.
  */
 public class ExampleSubsystem extends Subsystem {
-  // Put methods for controlling this subsystem
-  // here. Call these from Commands.
+  public static void Manual (TalonSRX starboard, TalonSRX port, Joystick driverStick)
+  {
+    
+    Double thro = driverStick.getRawAxis(1);
```

Comment:
Good work, variables look nice and have appropriate names. Try adding in code for a joystick, you can use the code in joe-drivetrain as an example.
The axis for yaw is 2.